{
  "Header": {
    "Original": "CVE-2021-20178 (A flaw was found in ansible module where credentials are disclosed in  ...)",
    "ID": "CVE-2021-20178",
    "Description": "(A flaw was found in ansible module where credentials are disclosed in  ...)"
  },
  "Annotations": [
    {
      "Original": "{DLA-3695-1}",
      "Type": "xref",
      "Bugs": [
        "DLA-3695-1"
      ]
    },
    {
      "Original": "- ansible 5.4.0-1 (bug #985753)",
      "Type": "package",
      "Package": "ansible",
      "Kind": "fixed",
      "Version": "5.4.0-1",
      "BugNo": 985753
    },
    {
      "Original": "[bullseye] - ansible \u003cnot-affected\u003e (Vulnerable code not present)",
      "Type": "package",
      "Release": "bullseye",
      "Package": "ansible",
      "Kind": "not-affected",
      "Description": "Vulnerable code not present"
    },
    {
      "Original": "[stretch] - ansible \u003cend-of-life\u003e (EOL'd for stretch)",
      "Type": "package",
      "Release": "stretch",
      "Package": "ansible",
      "Kind": "end-of-life",
      "Description": "EOL'd for stretch"
    },
    {
      "Original": "NOTE: https://bugzilla.redhat.com/show_bug.cgi?id=1914774",
      "Type": "NOTE",
      "Description": "https://bugzilla.redhat.com/show_bug.cgi?id=1914774"
    },
    {
      "Original": "NOTE: https://github.com/ansible-collections/community.general/pull/1621",
      "Type": "NOTE",
      "Description": "https://github.com/ansible-collections/community.general/pull/1621"
    },
    {
      "Original": "NOTE: https://github.com/ansible-collections/community.general/commit/3560aeb12f7061bf21d63ca0e1e19feb99c57de3",
      "Type": "NOTE",
      "Description": "https://github.com/ansible-collections/community.general/commit/3560aeb12f7061bf21d63ca0e1e19feb99c57de3"
    }
  ]
}